GVFS/GVFS.Common/Git/DiffTreeResult.cs

```diff
@@ -7,9 +7,15 @@ namespace GVFS.Common.Git
 {
     public class DiffTreeResult
     {
-        public const string TreeMarker = " tree ";
-        public const string BlobMarker = " blob ";
-        public const string CommitMarker = " commit ";
+        public const string TreeMarker = "tree ";
+        public const string BlobMarker = "blob ";
+
+        /// <summary>

```

Comment:
A one line comment that shows the format of the line would be more useful here, I think. 5 lines of comments to just say "the type is at position 7" doesn't tell me any more information than the constant declaration itself.